cmd/kubestellar-scheduler/cmd/scheduler.go

```diff
@@ -39,7 +39,11 @@ import (
 	scheduleroptions "github.com/kubestellar/kubestellar/cmd/kubestellar-scheduler/options"
 	edgeclientset "github.com/kubestellar/kubestellar/pkg/client/clientset/versioned/cluster"
 	edgeinformers "github.com/kubestellar/kubestellar/pkg/client/informers/externalversions"
+
 	"github.com/kubestellar/kubestellar/pkg/scheduler"
+
+	edgev1alpha1 "github.com/kubestellar/kubestellar/pkg/apis/edge/v1alpha1"
```

Comment:
nit: maybe remove all those spaces between the import lines ?

Reply:
`make imports` should be used to normalize import sections.  IMHO it is an error that CI does not check that this has been done.